packages/launcher-app/src/app/config.ts

```diff
@@ -45,7 +45,7 @@ function getAuthConfig(authMode: string): KeycloakConfig | OpenshiftConfig | und
           url: requireEnv(process.env.REACT_APP_OAUTH_OPENSHIFT_URL, 'openshiftOAuthUrl'),
           validateTokenUri: `${requireEnv(process.env.REACT_APP_LAUNCHER_API_URL, 'launcherApiUrl')}/services/openshift/user`,
         },
-        gitProvider: requireEnv(process.env.REACT_APP_GIT_PROVIDER, 'gitProvider') === 'github' ? 'github' : 'gitea',
+        gitProvider: (getEnv(process.env.REACT_APP_GIT_PROVIDER, 'gitProvider') || 'github') === 'github' ? 'github' : 'gitea',
```

Comment:
`getEnv(process.env.REACT_APP_GIT_PROVIDER, 'gitProvider') || 'github'` should be enough I guess.

Reply:
the type of the field is: `gitProvider: 'gitea' | 'github'` so we are forced to make sure it's one of these values